### PR TITLE
Minor edits to the step-51 introduction.

### PR DESCRIPTION
**Pull request by bangerth**
Specifically, make sure 'FE_DGPMonomial' is spelled correctly. While there,
also address a couple of other things.

### CODE REVIEW

File: examples/step-51/doc/intro.dox [hunk 1]
```diff
@@ -41,8 +40,15 @@ The HDG method achieves
 this goal by formulating the mathematical problem using Dirichlet-to-Neumann
 mappings.  The partial differential equations are first written as a first
 order system, and each field is then discretized via a DG method.  At this
-point the  single-valued "trace" values on the skeleton of the
+point, the  single-valued "trace" values on the skeleton of the
 mesh, i.e. element faces, are taken to be independent unknown quantities.
+This yields unknowns in the discrete formulation that fall into two categories:
+- Face unknowns that only couple with the unknowns from both sides of the face;
+- Cell unknowns that only couple with each other and the other unknowns
+  defined within the same cell. Crucially, no cell interior degree of freedom
```
Comment (kronbichler) [inline examples/step-51/doc/intro.dox]: Maybe reword this slightly: "Cell unknowns that only couple with the cell and face unknowns defined within the same cell."

**Comment by bangerth** [comment]
Wording fixed.
